web/src/app/administration/components/access-items-management/access-items-management.component.spec.ts

```diff
@@ -109,7 +112,10 @@ describe('AccessItemsManagementComponent', () => {
         { provide: FormsValidatorService, useClass: formValidatorServiceSpy },
         { provide: NotificationService, useClass: notificationServiceSpy },
         RequestInProgressService,
-        ClassificationCategoriesService
+        ClassificationCategoriesService,
+        StartupService,
+        TaskanaEngineService,
+        WindowRefService
```

Comment:
Why do we need this now?

Reply:
Because startupService uses those services